alpine-nix-rails-nginx/etc/runit/1

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+source /etc/profile.d/nix.sh
+
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
+IFS=$'\n\t'
```

Comment:
No, unfortunately the nix.sh (that is generated by nix) blows up.

I suppose it might be possible to push a fix upstream though, I could look at that in my free time.